auth/postgres/database.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package postgres
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/opentracing/opentracing-go"
+)
+
+var _ Database = (*database)(nil)
+
+type database struct {
+	db *sqlx.DB
+}
+
+// Database provides a database interface
```

Comment:
Is it possible to move this to tracing package ? I would expect to see something else in database package, file named database.go